spec/forms/user_piv_cac_setup_form_spec.rb

```diff
@@ -103,6 +103,18 @@
       end
     end
 
+    context 'when piv cac is required' do
+      let(:token) { 'good-token' }
+
+      it 'returns FormResponse with success: true when the token has an eku' do
+        expect_results_with_eku(true)
```

Comment:
IMO this kind of method adds a lot of indirection and makes things more inflexible in the future.

If it were 100% up to me I'd do what I outlined here: https://github.com/18F/identity-idp/pull/4005#discussion_r465227074

But this is not the hill I die on, so LGTM